avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastSerdeCache.java

```diff
@@ -170,7 +170,9 @@ public static FastSerdeCache getDefaultInstance() {
                   classPath += ":" + lib.getAbsolutePath();
                 }
               }
-              LOGGER.info("Inferred class path: " + classPath);
+              if (LOGGER.isDebugEnabled()) {
+                LOGGER.debug("Inferred class path: " + classPath);
```

Comment:
if you use {} you could drop the isDebugAEnabled() call with no penalty

Reply:
Fixed, thanks for the suggestion!